diff --git a/rtmidi_c.h b/rtmidi_c.h
index 82db105..fa25e2b 100644
--- a/rtmidi_c.h
+++ b/rtmidi_c.h
@@ -85,9 +85,9 @@ enum RtMidiErrorType {
 
 /*! \brief The type of a RtMidi callback function.
  *
- * \param timeStamp   The time at which the message has been received.
- * \param message     The midi message.
- * \param userData    Additional user data for the callback.
+ * param timeStamp   The time at which the message has been received.
+ * param message     The midi message.
+ * param userData    Additional user data for the callback.
  *
  * See \ref RtMidiIn::RtMidiCallback.
  */
@@ -129,7 +129,7 @@ RTMIDIAPI void rtmidi_error (enum RtMidiErrorType type, const char* errorString)
 
 /*! \brief Open a MIDI port.
  *
- * \param port      Must be greater than 0
+ * \param portNumber      Must be greater than 0
  * \param portName  Name for the application port.
  *
  * See RtMidi::openPort().
@@ -166,8 +166,10 @@ RTMIDIAPI int rtmidi_get_port_name (RtMidiPtr device, unsigned int portNumber, c
 
 /* RtMidiIn API */
 
-//! \brief Create a default RtMidiInPtr value, with no initialization.
-RTMIDIAPI RtMidiInPtr rtmidi_in_create_default (void);
+/*! \brief Create a default RtMidiInPtr value, with no initialization.
+ */
+
+RTMIDIAPI RtMidiInPtr rtmidi_in_create_default ();
 
 /*! \brief Create a  RtMidiInPtr value, with given api, clientName and queueSizeLimit.
  *
@@ -217,8 +219,10 @@ RTMIDIAPI double rtmidi_in_get_message (RtMidiInPtr device, unsigned char *messa
 
 /* RtMidiOut API */
 
-//! \brief Create a default RtMidiInPtr value, with no initialization.
-RTMIDIAPI RtMidiOutPtr rtmidi_out_create_default (void);
+/*! \brief Create a default RtMidiInPtr value, with no initialization.
+ */
+
+RTMIDIAPI RtMidiOutPtr rtmidi_out_create_default ();
 
 /*! \brief Create a RtMidiOutPtr value, with given and clientName.
  *
